tests/common/util/mailjet_connector_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the Email utility."""
+import tempfile
+import unittest
+from base64 import b64decode
+from unittest.mock import patch, call
+from urllib.error import HTTPError
+
+from google.cloud.forseti.common.util.email.mailjet_connector import MailjetConnector, Attachment
+from google.cloud.forseti.common.util.errors import EmailSendError
+from tests.unittest_utils import ForsetiTestCase
+
+
+class MailjetConnectorTest(ForsetiTestCase):
+    """Tests for the Email utility."""
+
+    def setUp(self):
```

Comment:
We should make the test skippable if the mailject is not installed/enabled.

https://github.com/forseti-security/forseti-security/pull/2317/files#diff-1916e18675909bcb2aa3083dc5c74917R34-R36

![image](https://user-images.githubusercontent.com/6977544/66611717-42be1d00-eb74-11e9-937a-64d3aa32d7fe.png)

Reply:
done